pom.xml

```diff
@@ -110,7 +110,7 @@
         <module>adapter-kafka</module>
         <module>adapter-amqp</module>
         <module>adapter-jdbc</module>
-        <module>adapter-pgvector</module>
+<!--        <module>adapter-pgvector</module>-->
```

Comment:
Should we also remove the `adapter-pgvector` sub-project altogether now that the full support is already available here?

Reply:
I'm going to add this in a TBD list for future work